R/QC3d.R

```diff
@@ -19,21 +19,21 @@
 #'
 
 
-QC3d <- function(d_metingen, d_parameter, verbose = F) {
+QC3d <- function(d_metingen, d_parameter, geleidendheid_veld_naam, verbose = F) {
```

Comment:
Je hebt nu een extra parameter toegevoegd. Die parameter moet ook beschreven worden in de manual (de 'roxygen' code boven de functie. Daar moeten een @param geledenheid_veld_naam toegevoegd worden. Zie https://r-pkgs.org/man.html#man-functions

Het is ook slim om geleidendheid_veld_naam alvast een 'default' waarde te geven van "GLDHD_VELD", dus dan wordt de functie regel:

```r
QC3d <- function(d_metingen, d_parameter, geleidendheid_veld_naam = "GLDHD_VELD", verbose = F)
```